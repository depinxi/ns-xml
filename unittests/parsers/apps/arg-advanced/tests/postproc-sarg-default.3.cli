-F "" -- "Set an empty value to an argument that defines a default value"