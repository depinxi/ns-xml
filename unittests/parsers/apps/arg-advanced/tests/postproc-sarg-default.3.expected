CLI: "-F", "", "--", "Set an empty value to an argument that defines a default value"
Value count: 1
Values: "Set an empty value to an argument that defines a default value"
Error count: 0
Subcommand: 
argumentF=
argumentFdefault=
argumentG=
multiArgumentH=
multiArgumentI=
